website/src/pages/jobs.tsx

```diff
@@ -316,6 +309,31 @@ const CareersPage = props => {
                     </Carousel.Item>
                 </Carousel>
 
+                <Carousel
+                    className="reviews-carousel"
+                    prevIcon={<ArrowLeftIcon color="#808080" />}
+                    nextIcon={<ArrowRightIcon color="#808080" />}
+                >
+                    <Carousel.Item>
+                        <img src="../careers/review1.png" />
```

Comment:
I know this page already contains a lot of repetitive manual markup that could be abstracted into components and used in loops, but as an improvement, you could refactor and simplify this given the url structure where the assets live to something like this:

```jsx
{new Array(5).fill(null).map((el, i) => (
    <Carousel.Item key={`review-${i}`}>
        <img src={`/careers/review${i}.png`} />
    </Carousel.Item>
))}
```

You'll have to rename your assets to start at `0` given the loop.

Also, this is out of scope, but feel free to change all of our urls on this page to be absolute paths instead of relative paths since assets are served from the root directory. Notice Gatsby builds everything under `public` when you run `yarn build` 🤓

Reply:
Ah, I was thinking of doing a loop but bypassed it. Thanks!

Reply:
Ah, I was thinking of doing a loop but bypassed it. Thanks!